commands/file_uploader_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("FileUploader", func() {
 					uploadedFiles, status = fileUploader.UploadFiles()
 				})
 				ex.ExpectSuccess(status.ToInt(), output)
-				Expect(uploadedFiles).To(Equal([]*models.FileMetadata{}))
+				Expect(uploadedFiles).To(BeNil())
```

Comment:
Why are these expected to be nil?

Reply:
Because when I changed it from 
`filesToUpload := []os.File{}`
to 
`var filesToUpload []os.File` 
the default value is nil, not an empty array